keyboards/hineybush/h88/rules.mk

```diff
@@ -0,0 +1,32 @@
+# MCU name
+MCU = atmega32u4
+
+
+# Interrupt driven control endpoint task
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
Do you know what Boot loader is being used?  

If so, could you specify that rather than the size?  Eg: 
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   atmega32a    bootloadHID
BOOTLOADER = atmel-dfu
```

If you're not sure, then it's most like atmel-dfu.